docs/specifications/tests/Consistency-TP/consistency01.md

```diff
@@ -117,6 +122,7 @@ When comparing SOA serial it must be done using the arithmetic defined in
 
 None
 
+[Basic04]:                    ../Basic-TP/basic04.md
```

Comment:
The indentation here is off.

There are more of these below as well as in the sibling pull requests.

Reply:
When the link section was not formatted I just ignored the formatting. I have now sorted and formatted the link sections in the specifications in this PR.

@mattias-p, please re-review.